src/kOS/Suffixed/CrewMember.cs

```diff
@@ -7,7 +7,7 @@ namespace kOS.Suffixed
     [kOS.Safe.Utilities.KOSNomenclature("CrewMember")]
     public class CrewMember : Structure
     {
-        private readonly ProtoCrewMember crewMember;
+        public readonly ProtoCrewMember crewMember;
```

Comment:
Did you mean to change this or was it just for debug.

Reply:
My changes to kos-career use it, but I ended up just using krafs publicizer so that I didn't have to wait for this to be merged.  So I don't really care if that change is included, but it might be nice as a marker to say that external code is now relying on it.  Or maybe it would be better to have a public get/private set property.